ambry-api/src/main/java/com.github.ambry/config/ServerConfig.java

```diff
@@ -61,6 +61,13 @@
   @Default("false")
   public final boolean serverEnableStoreDataPrefetch;
 
+  /**
+   * Implementation class for StoreKeyConverterFactory
+   */
+  @Config("server.store.key.converter.factory.class")
```

Comment:
minor: we (mostly) don't use `.class` at the end for factory config names. (`server.store.key.converter.factory`)

Reply:
Got rid of the ".class"